pkg/controller/admissionchecks/multikueue/workload.go

```diff
@@ -198,11 +198,11 @@ func (w *wlReconciler) Reconcile(ctx context.Context, req reconcile.Request) (re
 	}
 
 	managed, unmanagedReason, err := adapter.IsJobManagedByKueue(ctx, w.client, types.NamespacedName{Name: owner.Name, Namespace: wl.Namespace})
-	if err != nil {
+	if err != nil && !isDeleted {
```

Comment:
This will conflict with https://github.com/kubernetes-sigs/kueue/pull/2355. We can probably revert it from here

Reply:
sure ,  I found 2355 while working on this.

Reply:
rebased